test/libsolidity/semanticTests/getters/struct_with_bytes_simple.sol

```diff
@@ -0,0 +1,17 @@
+contract C {
+    struct S {
+        uint a;
+        bytes b;
+        mapping(uint => uint) c;
+        uint[] d;
+    }
+    uint shifter;
```

Comment:
Unused?

Reply:
it is supposed to shift the storage slot so that it is not just zero

Reply:
Yeah, right, just realized that, but was too late in editing the comment :-).